tensorflow/core/kernels/sparse_to_dense_op_gpu.cu.cc

```diff
@@ -192,8 +196,13 @@ void LaunchSparseToDense<T, Index>::operator()(
       // Ensure that within the callback, the proper GPU settings are
       // configured.
       auto stream = c->op_device_context()->stream();
+#if GOOGLE_CUDA
       se::cuda::ScopedActivateExecutorContext scoped_activation{
           stream->parent()};
+#else
+      se::gpu::ScopedActivateExecutorContext scoped_activation{
```

Comment:
Can we just use this branch for both?

Reply:
you are right, that should work....made the suggested and pushed it out.